### PR TITLE
S2t2

### PR DESCRIPTION
**Pull request by Phildallen**

### CODE REVIEW

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Indentation

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: too many blank end spaces

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Posted needs to be converted from International date format into UK date

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Buttons look a bit too round compared to the brief. Could they be more rectangular?

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: semi colons at end of lines - consistency - haven't always used them

**Review by JamesMcGraaw — CHANGES_REQUESTED** [review]
Looks good, a few more changes need to be made before approval

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Needs to go under title on mobile

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Gets warped shape on mobile view

File: src/components/DetailModal/index.js
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js:66]: this text could all be a little smaller, especially looking on mobile

File: src/components/DetailModal/index.js
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

**Pull request closed without merging**
